README.md

```diff
@@ -76,20 +76,41 @@ follow the prompts questions, provide `myvoltoproject` as project name then, whe
 
 We recommend Plone as backend of choice for Volto.
 
-You can bootstrap a ready Docker Plone container with all the dependencies and ready for Volto use:
+You can bootstrap a ready Docker Plone container with all the dependencies and ready for Volto use. We recommend to use the new Plone docker builds based in `pip` [plone/plone-backend](https://github.com/plone/plone-backend) image:
```

Comment:
I'd say formulate it as "use the pip-based Plone docker builds"... at some point they're not gonna be new.

Reply:
Done